Bottom: 1053044e6fcae5246cf14d5f921d74d49154cc4d
Top:    2a7ad340cbc88b9e7a84da21157945495b9db169
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-21 05:41:16 +0000

Avoid clearing pins / vias from polygons... it is already done!




---

diff --git a/src/draw.c b/src/draw.c
index 2b81861..bfda63a 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1239,6 +1239,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1261,6 +1262,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
